tmva/sofie/inc/TMVA/ROperator_Softmax.hxx

```diff
@@ -58,199 +58,108 @@ public:
          throw std::runtime_error("TMVA SOFIE Operator Softmax called to Generate without being initialized first");
       }
       std::stringstream out;
-      int size = fShape.size();
-      auto length = ConvertDimShapeToLength(fShape);
-      auto stride = UTILITY::ComputeStrideFromShape(fShape);
+      size_t size = fShape.size();
+      auto length_str = ConvertDimShapeToLength(fShape);
+      size_t length = std::stoul(length_str);
```

Comment:
you don't need to convert to an integer, since in some case it might fail : e.g length  = " N * 32"